resources/migrations/20220114122744-remove-owner-modifier.down.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE resource
+ADD COLUMN ownerUserId varchar(255),
+ADD COLUMN modifierUserId varchar(255);
```

Comment:
i think PSQL lowercases the column names by default, but maybe we could use the same lowercase naming as in `up` migration for consistency?